src/main/java/frc/robot/Constants.java

```diff
@@ -155,12 +155,28 @@ public static class TurretConstants {
     public static final int highGearCAN_CODER_ID = 0;
     public static final int lowGearCAN_CODER_ID = 0;
     public static final double ENCODER_ZERO_OFFSET_FROM_TURRET_ZERO_REV = 0;
+    public static final double GEAR_RATIO =
+        6.8333; // (24:164)  0.25 for the revbot prototype turret
+
+    // Simulation Vals
+    public static final double TURRET_MOI = 1; // TODO: Get real num // Moment of Inertia
+    public static final double TURRET_RADIUS = Units.inchesToMeters(12);
   }
 
   public static class OperatorConstants {
     public static final int kDriverControllerPort = 0;
   }
 
+  public static class SensorConstants {
+    // TODO: Get real ids
+    public static final int SOURCE_BREAK_ID = 1;
```

Comment:
I think there are one too many sensors here.

We have a plan for one sensor on the intake.
One sensor between the trigger rollers and the shooter rollers

And one sensor on the amp arm.